ext/native/util/text/utf8.cpp

```diff
@@ -477,9 +477,9 @@ std::string ConvertWStringToUTF8(const std::wstring &wstr) {
 	return s;
 }
 
-static size_t ConvertUTF8ToWStringInternal(wchar_t *dest, size_t destSize, const std::string &source) {
-	const wchar_t *const orig = dest;
-	const wchar_t *const destEnd = dest + destSize;
+static size_t ConvertUTF8ToUCS16Internal(char16_t *dest, size_t destSize, const std::string &source) {
+	const char16_t *const orig = dest;
+	const char16_t *const destEnd = dest + destSize;
 
 	UTF8 utf(source.c_str());
 
```

Comment:
We shouldn't check `sizeof(wchar_t) here anymore, then, right?

-[Unknown]

Reply:
Indeed, oops.

Reply:
Oops, my bad.

-[Unknown]